packages/amplify_auth_cognito/example/ios/unit_tests/amplify_auth_cognito_tests.swift

```diff
@@ -1389,7 +1389,7 @@ class amplify_auth_cognito_tests: XCTestCase {
                 // all tokens should be present with userpool-only access
                 XCTAssertEqual(3, (res.toJSON()["tokens"] as?  [String: String])!.count)
                 // credentials map should be empty
-                XCTAssertEqual(0, (res.toJSON()["credentials"] as?  [String: String])!.count)
+                XCTAssertEqual(nil, (res.toJSON()["credentials"] as?  [String: String]))
```

Comment:
Nit: Use XCTAssertNil instead?
```suggestion
                XCTAssertNil(res.toJSON()["credentials"] as? [String: String])
```